credentials/generate-revocation-set.py

```diff
@@ -64,6 +64,31 @@ def extract_single_integer_attribute(subject, oid):
     return None
 
 
+def extract_single_attribute_from_cn(cn, marker):
+    val_len = 4
+    start_idx = cn.find(marker)
+
+    if start_idx != -1:
+        val_start_idx = start_idx + len(marker)
+        return int(cn[val_start_idx:val_start_idx + val_len], 16)
```

Comment:
Extract the substring before calling `int` and return `None` if the substring length `!= len('FFFF')`. Current code can fail on a string ending in `Mvid:` or if there are insufficient characters for the fallback encoding.